### PR TITLE
Frm main edits

### PR DESCRIPTION
**Pull request by AlexSananka**
@dannyparsons I have disabled the summary and graph options on threeVariable frequencies as suggested by @rdstern 

This  is ready to be merged.

### CODE REVIEW

File: instat/dlgOneVarCompareModels.vb [hunk 1]
```diff
@@ -1,106 +1,119 @@
...
+            cmdDisplayObjects.Enabled = True
+        End If
+        TestOKEnabled()
+        '        sdgOneVarCompareModels.SetModelFunction(ucrBase.clsRsyntax.clsBaseFunction)
+    End Sub
+
+    Private Sub cmdDisplayObjects_Click(sender As Object, e As EventArgs) Handles cmdDisplayObjects.Click
+        '        sdgOneVarCompareModels.SetRFunction(ucrBase.clsRsyntax.clsBaseFunction, bResetSubdialog)
+        'bResetSubdialog = False
+        'sdgOneVarCompareModels.ShowDialog()
```
Comment (dannyparsons) [inline instat/dlgOneVarCompareModels.vb]: What's going on here? Why is nothing happening when this is clicked. And why are these changes in this branch?

Comment (AlexSananka) [inline instat/dlgOneVarCompareModels.vb]: Wait,I at some point merged Lily's onevarcomparemodel branch to my master branch.
@dannyparsons Could this be the issue?

Comment (dannyparsons) [inline instat/dlgOneVarCompareModels.vb]: Yes that would be why. You should never merge anything into your master other than updating from the main branch. Now your master is out of sync with the main repo.

Comment (AlexSananka) [inline instat/dlgOneVarCompareModels.vb]: I did this by mistake,I think I'll have to redo this in a different branch.All the same this are very tiny edits.

**Pull request closed without merging**